### DISCUSSION TITLE
[QUESTION] Is the Safety Score data Tesla is gathering accessible in the API?

### DISCUSSION THREAD

**Discussion by SemoTech** [teslamate-org/teslamate#2011] [Q&A]
@adriankumpf with the advent of the "button" for requesting admission in the FSD Beta program, Tesla is now gathering a LOT of driver behavior data, and displays it in the Tesla App as an aggregated "Safety Score".  They then provide a summary breakdown for key indicators by day and trip. 

Does the API present any of this data, and can it be logged along with all the other data TeslaMate logs?

Thank you.

**Comment by LelandSindt** [comment]
@SemoTech  Safety Score is being looked into..  :)

https://github.com/timdorr/tesla-api/discussions/444
https://github.com/adriankumpf/teslamate/discussions/1997

**Comment by SemoTech** [comment]
Music to my ears @LelandSindt 
Thanks for flagging/chasing. 

Meantime hope the SSO auth does not cause any grief.
